modules/cluster-logging-deploy-cli.adoc

```diff
@@ -0,0 +1,406 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-cli_{context}"]
+= Installing cluster logging using the CLI
+
+[role="_abstract"]
+You can use the {product-title} CLI to install the Elasticsearch and Cluster Logging operators.
+
+.Prerequisites
+
+* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
+requires its own storage volume.
++
+Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster rather than increasing the memory on existing nodes.
+
+.Procedure
+
+To install the Elasticsearch Operator and Cluster Logging Operator using the CLI:
+
+. Create a Namespace for the Elasticsearch Operator.
```

Comment:
Namespace creation is not something that's available to dedicated-admins.

@RiRa12621 was this something that you discovered to be a necessary step?

Reply:
I think it's in the OCP docs, but we're doing that for OSD via https://github.com/openshift/managed-cluster-config/blob/de782b19a56816c647675994d1b53c1073d6c517/deploy/osd-openshift-operators-redhat/00-namespace.yaml

Reply:
Ok, so then this part about namespace is correct per Rick, so I'm resolving this conversation and asking for QE verification.